apps/web/src/query/user.ts

```diff
@@ -24,6 +25,8 @@ export const getUserById = cache(async (id: string) => {
             password: "demo",
             isAdmin: false,
             created: new Date(),
+            isActive: true,
+            appVersion: Versions.support,
```

Comment:
```suggestion
            appVersion: Versions.transparency,
```
Wir hatten doch gesagt, Demo User soll nur die Basic Funktionen haben?

Reply:
Jo, hab ich zu mtesten umgestellt